CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+## How to contribute code
+
+Follow these steps to submit your code contribution.
+
+### Step 1. Open an issue
+
+Before making any changes, we recommend opening an issue (if one doesn't already
+exist) and discussing your proposed changes. This way, we can give you feedback
+and validate the proposed changes.
+
+If the changes are minor (simple bug fix or documentation fix), then feel free
+to open a PR without discussion.
+
+### Step 2. Make code changes
+
+To make code changes, you need to fork the repository. You will need to setup a
```

Comment:
I think we should link to a style guide or mention formatting somewhere here (fine if that's a follow up).

Reply:
Let's do this in a follow up as that's a larger task.